### PR TITLE
Auth fixes?

### PR DESCRIPTION
**Pull request by JamesJeffryes**
So I don't know if this fixes the root cause of the issues when called by GFU in CI but getting the local tests to run is probably progress

### CODE REVIEW

File: kbase.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ service-language:
     java
 
 module-version:
-    0.0.2
+    0.0.3
 
 owners:
     [rsutormin]
```
Comment (MrCreosote) [inline kbase.yml]: Add yourself as an owner so you can release it...

File: deploy.cfg [hunk 2]
```diff
@@ -4,4 +4,8 @@ job-service-url = {{ job_service_url }}
 workspace-url = {{ workspace_url }}
 shock-url = {{ shock_url }}
 handle-service-url = {{ kbase_endpoint }}/handle_service
+srv-wiz-url = {{ srv_wiz_url }}
```
Comment (MrCreosote) [inline deploy.cfg]: Does it use the service wizards or njsw? I don't think so...

**Review by MrCreosote — COMMENTED** [review]
Looks good, just a couple of minor things.

File: deploy.cfg (hunk 2 above)
Comment (JamesJeffryes) [reply deploy.cfg]: Nope, these are just part of the boilerplate for deploy.cfg now. Can remove.

File: kbase.yml (hunk 1 above)
Comment (JamesJeffryes) [reply kbase.yml]: Have catalog admin powerz and I don't _really_ want to have my name on a Java package but I guess I should...

**Comment by JamesJeffryes** [comment]
ok @MrCreosote, should be GTG now.

**Review by MrCreosote — APPROVED** [review]
Excellent. You are now responsible for this module forever and always

**Pull request merged**